.github/workflows/espresso-docker.yml

```diff
@@ -152,15 +152,17 @@ jobs:
           username: ${{ github.repository_owner  }}
           password: ${{ secrets.GITHUB_TOKEN  }}
 
-      - name: Remove digests dir
-        run: |
-          rm -rf "${{ runner.temp }}/digests"
+      - name: Download digest amd64
+        uses: actions/download-artifact@v4
+        with:
+          name: "${{ matrix.target }}-digests-${{ github.run_number }}-amd64"
+          path: "${{ runner.temp }}/${{ matrix.target }}-${{ github.run_number }}-digests"
 
-      - name: Download digests
-        uses: actions/download-artifact@v3
+      - name: Download digest arm64
```

Comment:
We're now doing an amd64 and arm64 build? Sounds good to me!

Reply:
yea  we had to do this because of this:  https://github.com/actions/upload-artifact?tab=readme-ov-file#not-uploading-to-the-same-artifact